Tests/kaas/kaas-sonobuoy-tests/Makefile

```diff
@@ -54,6 +54,7 @@ dev-prerequests:
 	@jq --version
 	@go version
 	@docker buildx version
+	# @golangci-lint version --format short	
```

Comment:
I would say, we can uncommented this and apply it here as a prerequisite.

Reply:
Or should we remove it? It's unnecessary because I added logic that automatically checks if `golangci-lint` is installed every time you run the `make lint` command. If it's not installed, it will automatically install version `1.61.0` for you.

Reply:
What do you think @tonifinger, should I remove it or just uncommented it?

Reply:
I think your approach of installing the linter automatically is good.
I agree it can be removed.

Reply:
Thanks, I removed that comment in https://github.com/SovereignCloudStack/standards/pull/771/commits/8e69fa487a25311da1d839d2b5d59c7dbda7ad4a